added unit test cases

ramkishor-ch opened the pull request:
Hi @allmightyspiff,

I have written the unit test cases, tested it and got the 100% code coverage.
Please review it and approve this pull request.

Please find the details.

Details:

Title: 
Get code coverage of the following file to 100%.
CodeCoverage: SoftLayer\CLI\account\events.py

Description:
I have added four **unit test cases** of these functions: test_account_events(), test_account_planned_events(), test_account_unplanned_events(), test_account_announcement_events() in /softlayer-python/tests/CLI/modules/**account_tests.py** to get code coverage of 100% for this file:SoftLayer/CLI/account/events.py.

Linked Issues: #1935

Please find the screen shots for reference.

Screen Shot 1:
<img width="1479" alt="Coverage_1_Screenshot 2023-05-10 at 12 00 41 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/3e1d15d6-d70b-44ad-83dd-313c29a09df2">



Screen Shot 2:
<img width="1476" alt="Coverage_2_Screenshot 2023-05-10 at 12 03 26 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/6b060412-e162-4b58-b8e4-8407b83e53a4">



Screen Shot 3:
<img width="1481" alt="Coverage_3_Screenshot 2023-05-10 at 12 05 53 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/d5e5221e-c37a-4af0-90e0-b4108e3550d3">



Screen Shot 4:
<img width="1479" alt="Coverage_4_Screenshot 2023-05-10 at 12 06 20 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/14d1409f-9ff1-4418-97b7-136b20cc7981">



Screen Shot 5:
<img width="993" alt="Coverage_5_100_Percent_Screenshot 2023-05-10 at 12 09 19 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/6b262df4-5e2a-4a16-aef8-01a36cfe9959">



Screen Shot 6:
<img width="1472" alt="Screenshot 2023-05-10 at 11 45 35 AM" src="https://github.com/softlayer/softlayer-python/assets/125332006/03c00b3b-2bea-485a-8f8f-d6dd8794306a">


Thanks,
Ramkishor Chaladi.

allmightyspiff commented:
Looks like the `analysis` test failed. To see the results locally you can run `tox -e analysis`

```
tests/CLI/modules/account_tests.py:[20](https://github.com/softlayer/softlayer-python/actions/runs/4934511236/jobs/8819642145?pr=1938#step:5:21)3:1: W293 blank line contains whitespace
tests/CLI/modules/account_tests.py:208:1: W293 blank line contains whitespace
tests/CLI/modules/account_tests.py:[21](https://github.com/softlayer/softlayer-python/actions/runs/4934511236/jobs/8819642145?pr=1938#step:5:22)3:1: W293 blank line contains whitespace
tests/CLI/modules/account_tests.py:217:61: W292 no newline at end of file
```

allmightyspiff reviewed (changes requested):
Code looks good, just need to fix the analysis issues. Thanks

ramkishor-ch commented:
Hi @allmightyspiff,

I have fixed the analysis test.
Please **review** it and **approve** this pull request.

Please find the details.

Details:

Title:
Get code coverage of the following file to 100%.
CodeCoverage: SoftLayer\CLI\account\events.py

Description:
I ran **tox -e analysis** command in my local system and fixed the analysis test, now it is **working fine**.

Linked Issues: https://github.com/softlayer/softlayer-python/issues/1935

Please find the screen shots for reference.

Screen Shot 1:
<img width="1476" alt="tox_analysis_Screenshot_1 2023-05-11 at 11 46 58 AM" src="https://github.com/softlayer/softlayer-python/assets/125332006/47e5ba18-ca3e-4999-8ec2-a5bd59f686d0">


Screen Shot 2
<img width="1507" alt="tox_analysis_Screen_Shot_2 2023-05-11 at 2 19 14 PM" src="https://github.com/softlayer/softlayer-python/assets/125332006/5540be7b-ee52-4ee4-80be-649e7318cdba">


Thanks,
Ramkishor Chaladi.
